src/filter/elastic_scale/elastic_scale.cpp

```diff
@@ -54,10 +54,10 @@ class ElasticScale : public frei0r::filter
     ElasticScale(unsigned int width, unsigned int height) : m_width(width), m_height(height)
 
     {
-        register_param(m_scaleCenter,"scaleCenter","Center position of the linearly scaled area");
-        register_param(m_linearScaleArea,"linearScaleArea","Area of the linearly scaled area");
-        register_param(m_linearScaleFactor,"linearScaleFactor","Amount how much the image is scaled");
-        register_param(m_nonLinearScaleFactor,"polynomialFactor","Polynomial factor for the transition between linear and non-linear scaling");
+        register_param(m_scaleCenter,"scaleCenter","Center position of the linear scaled area");
+        register_param(m_linearScaleArea,"linearScaleArea","Width of linear sclaed area");
+        register_param(m_linearScaleFactor,"linearScaleFactor","Amount how much the linearScaleArea is scaled");
+        register_param(m_nonLinearScaleFactor,"nonLinearScaleFactor","Amount how much the outer left and outer righter area are scaled non linearly");
```

Comment:
Please be aware that some applications use the name (scaleCenter, etc.) as the label in their generated UI. So, a non-identifier style name is more appropriate: Center, Linear Width, Linear Scale Factor, Non-linear Scale Factor

Reply:
Do you mean changing the respective internal member variables of the class, or just renaming the parameter names, those member variables are registered at by register_param?